code/game/objects/effects/effects.dm

```diff
@@ -13,4 +13,38 @@
 	return
 
 /obj/effect/attack_hulk(mob/living/carbon/human/user, does_attack_animation = FALSE)
-	return FALSE
\ No newline at end of file
+	return FALSE
+
+/obj/effect/decal
+	var/no_scoop = FALSE   //if it has this, don't let it be scooped up
+	var/no_clear = FALSE    //if it has this, don't delete it when its' scooped up
+	var/list/scoop_reagents = null
+
+/obj/effect/decal/Initialize(mapload)
```

Comment:
We really need to convert all atoms to use `Initialize`, it's starting to cause problems.

Reply:
yeah we do but i can't remeber if its due to the fact we don't have all the subsystems in..orrrr just no ones done it...